### ISSUE TITLE
UTF-8 decoding error

### ISSUE THREAD

**Issue by geovanidps** [skavngr/rapidscan#57]
![image](https://github.com/user-attachments/assets/c8bc1372-2796-41bc-9a0b-1215b710c878)

UTF-8 decoding error
The 'utf-8' codec cannot decode byte 0xe1 at position 12948, the continuation byte is invalid.

I corrected the following lines:

![image](https://github.com/user-attachments/assets/248e759c-7463-4bb0-8077-6fe583a7888d)

adding the following

_rs_tool_output_file = open(temp_file, errors='ignore').read()_

I also adjusted the time interval in the following:
![image](https://github.com/user-attachments/assets/234bfa59-408d-4f5f-9337-4cd7e1e0a783)

_def display_time(seconds, granularity=3):
    result = []
    seconds = seconds + 1
    for name, count in intervals:
        value = seconds // count
        if value > 0:
            seconds -= value * count
            if value == 1:
                name = name.rstrip('s')
            result.append('{} {}'.format(value, name))
    return ', '.join(result[:-1]) + ' and ' + result[-1] if len(result) > 1 else result[0]_

adding the following:

![image](https://github.com/user-attachments/assets/84f9c055-1cef-40d1-83af-9aa6a9d1d6bb)

_def scan_domain(domain):
    print(f"Scanning {domain}...")
    start_time = time.time()
    # Add your scanning logic here
    end_time = time.time()
    elapsed_time = end_time - start_time
    print(f"Scan completed in {display_time(elapsed_time)}")_

In summary, the _scan_domain_ function provides a basic structure for scanning a domain but man, the actual scanning logic needs to be implemented within the function.

I hope I helped with something
